### ISSUE TITLE
Right to Self-Defense and the Home...

### ISSUE THREAD

**Issue by thoth** [jonstokes/shootercontrol#12]
I would like to posit that there needs to be a provision for storage, relocation, and protection while in the home.

If you are currently unlicensed (but not a current/former felon), you should still maintain the ability to defend yourself in your home.  

Also, just because you are not able to "use/touch" them at ranges, competitions, hunting, etc, they still need to be safely stored, which I assume would be in the owner's home.  

Further I would suggest that they not be allowed to remove them from their home (if currently unlicensed) without some sort of notification.  For instance moving to a new home.  Basically a one use permit to move from point A to point B in direct a manner as possible (violations, of course, incurring the penalty).

**Comment by tsgsjeremy** [comment]
Great points. This is one of the 2A issues Jon pointed out. It would be a heavy lift constitutionally to preclude unlicensed non-felons from legally purchasing a firearm. Likewise, making the touching of a firearm by an unlicensed non-felon could be construed as an end run around the 2A.

That's why I proposed to leave government out of the picture and keep the network privately run. The government can't prevent you from buying/owning a firearm without running afoul of the constitution. But a private gun dealer can decline to sell to you, or a range can decline to let you use their facilities without issue. (See the Public Comment issue I posted for more info.)

I realize that this would leave a major loophole, but a) there are mitigation strategies, and b) I don't see any alternative without repeal of the 2A.

Possible mitigation factors
- Initially it would be a hard sell to convince dealers and ranges to only do business with network members. But bell curves work and at about 20% adoption the jump to 80% should move pretty quickly.
- Preserving a way of life. I don't believe I'm alone in thinking that given the world today it is only a matter of time before there's a serious and successful attempt to repeal the 2A. For those of us with such a mindset, we would welcome this type of voluntary proposal because it protects the 2A. It would make the "good guy with a gun" concept not just a concept, but a reality that anti-gunners could see with their own eyes.
- It might be tough to implement while still maintaining privacy, but transparency would also need to be a priority. Violators could be named and shamed. All networks, their structure, and membership counts could be made public. Basically everything short of member rolls should be open and accessible to everyone.
- Even though government and law enforcement wouldn't be able to charge and prosecute network violations, they could still report violations back to the network for adjudication.
- There would still be hold outs, but that could allow the authorities to be more efficient with their resources. Networks would theoretically be self-policing, so government authorities would have a smaller pond to focus on.

**Comment by nalanbar** [comment]
Without government involvement, this becomes a set of restrictions only,
without anything as a gain for the 2A side of the aisle. I very much doubt
you will get 5% adoption, much less 20%.
The carrot on the end of this stick is the relaxation of laws pertaining to
SBR's, suppressors, and (hopefully) CCW, etc. Without governmental
involvement, that all goes out the window.

**Comment by ghost** [comment]
This is slightly off-topic, but perhaps appropriate: 

I had an aunt who had a neighbor that she was providing hospice care to. The gentleman passed away and left everything he had to her (no next of kin)... This included an old pistol in the sock drawer (Still loaded, btw.)

There might be need for some exclusions in these strange circumstances...

**Comment by tsgsjeremy** [comment]
@nalanbar I agree it's a heavy lift sans government mandate. My main problem with going the government route is that it means the 2A will have to be curtailed. In that case the carrot becomes a mirage. In other words, if the government can create laws that deny you the right to own a gun in the first place, what difference does it make that there's no longer a ban on suppressors? You can't buy a gun to use it with. And if that carrot works and we weaken the 2A once, what's to prevent more and more erosion?

The border security debate is similar. We demand the government secure the border. They say "Sure! Right after we reform immigration." We reform immigration over and over, but somehow the border never seems to get secured.

**Comment by thoth** [comment]
I think that the government involvement would be somewhat similar to the process for buying now.  Instead of a Form 4473, you show your network card.  No card, no purchase.  And the card can be validated against the "block chain" idea to verify it is still valid.

Basically dealers are incentivized from the BATFE to only allow authorized people (and have a simple and fast way to vet them), but the "registry" is still decentralized.

**Comment by tsgsjeremy** [comment]
@thoth That's exactly the kind of thing I'm envisioning.

**Comment by nalanbar** [comment]
I think that solution works. The incentive for dealers and gun owners would
be no more tax stamps, and no more reciprocity barriers to travel, which
should boost sales. The incentive for the antis would be a self policing
groups empowered and incentivized to report to the police.

Which brings up another point, which may be worth its own discussion:
reporting.

**Comment by jonstokes** [comment]
I'm liking this direction better and better. It still removes the major carrot, i.e. relaxation of NFA rules, but it seems less crazy and more practical. In other words, we're farming out the background check part of the system to the shooter networks, and telling FFLs to only do business with licensed shooters.

We don't get the NFA goodies, but it's better than nothing.

Also, or what it's worth, I totally agree with this and feel the same way:

> Preserving a way of life. I don't believe I'm alone in thinking that given the world today it is only a matter of time before there's a serious and successful attempt to repeal the 2A. For those of us with such a mindset, we would welcome this type of voluntary proposal because it protects the 2A. It would make the "good guy with a gun" concept not just a concept, but a reality that anti-gunners could see with their own eyes.

We need to have a developed alternative ready at hand when a serious attempt on our rights is made.

**Comment by thoth** [comment]
@jonstokes what do you mean by "We don't get the NFA goodies"?

**Comment by jonstokes** [comment]
> @jonstokes what do you mean by "We don't get the NFA goodies"?

Or, maybe we do. I mean, I was envisioning the elimination of NFA restrictions on SBRs and suppressors and such as a tradeoff for making this regime mandatory for all legal access to firearms. But if we could get that stuff in exchange for making shooter networks the default BATF vetting/background-check mechanism, then that would certainly be awesome.

**Comment by tsgsjeremy** [comment]
Thanks for the forum Jon. Folks are bringing up good points that challenge the premise which further advances the project. And you keep throwing out concepts I've never thought of, yet they seem so obvious after the fact. Like having an answer to the question "Well what are YOU going to do about it?"

I feel like Salieri to your Mozart. LOL

**Comment by jonstokes** [comment]
I think this comment was actually meant for this issue, but it works in either one :)

https://github.com/jonstokes/shootercontrol/issues/4#issuecomment-228237613
